src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -32,19 +29,24 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Coordinate")
-@Description("Represents a given coordinate of a location. ")
-@Examples({"player's y-coordinate is smaller than 40:",
-		"	message \"Watch out for lava!\""})
-@Since("1.4.3")
-public class ExprCoordinate extends SimplePropertyExpression<Location, Number> {
+@Description("Represents a given coordinate of a location/chunk. ")
+@Examples({
+	"player's y-coordinate is smaller than 40:",
```

Comment:
Better for new users to understand quickly
```suggestion
	"if player's y-coordinate is smaller than 40:",
```